examples/example_nc.conf

```diff
@@ -35,6 +35,7 @@ Tmin = t_min
 prec = prec
 t_max = t_max
 t_min = t_min
+swe = swe
```

Comment:
Could you add this to the rest of the example input files?  They are broken without it.